UT4MasterServer.Services/Scoped/RatingsService.cs

```diff
@@ -0,0 +1,210 @@
+﻿using MongoDB.Driver;
+using UT4MasterServer.Common;
+using UT4MasterServer.Models;
+using UT4MasterServer.Models.DTO.Response;
+using UT4MasterServer.Models.DTO.Responses;
+
+namespace UT4MasterServer.Services;
+
+public sealed class RatingsService
+{
+	private readonly IMongoCollection<Rating> ratingsCollection;
+
+	public RatingsService(DatabaseContext dbContext)
+	{
+		ratingsCollection = dbContext.Database.GetCollection<Rating>("ratings");
+	}
+
+	public async Task<MMRBulkResponse> GetRatingsAsync(EpicID accountID, MMRBulkResponse mmrBulk)
+	{
+		var ratingTypes = mmrBulk.RatingTypes.Intersect(Rating.AllowedRatingTypes);
```

Comment:
If a request for some rating we don't support have comes in, we still want to return a default rating for specified rating type.

Reply:
Done.